File: patterns/c-patterns/pattern85.c
// Pattern 85: ...

// *
// **
// ***
// *
// **
// ***
// *
// **
// ***

int main()
{
    int n = 3;
    int i, j, k;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            for (k = 1; k <= j; k++)
            {
                printf("*");
            }
            printf("\n");
        }
    }

    return 0;
}

File: patterns/javascript-patterns/pattern85.js
// Pattern 85: ...

// *
// **
// ***
// *
// **
// ***
// *
// **
// ***

const n = 3;

for (let i = 1; i <= n; i++) {
    for (let j = 1; j <= n; j++) {
        let row = '';
        for (let k = 1; k <= j; k++) {
            row += '*';
        }
        console.log(row);
    }
}
